<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QWidget" name="Notepad">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenu {
	background-color: rgb(250, 250, 250);
	border-radius: 16px
}
QMenu:selected {
	color: rgb(64, 64, 64);
	background-color: rgb(224, 224, 224);
}
QMenu::separator {
	color: rgb(224, 224, 224)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QMenuBar" name="menubar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="defaultUp">
      <bool>false</bool>
     </property>
     <property name="nativeMenuBar">
      <bool>true</bool>
     </property>
     <widget class="QMenu" name="menuFile">
      <property name="title">
       <string/>
      </property>
      <addaction name="actionOpen"/>
      <addaction name="actionOpenFolder"/>
      <addaction name="actionSave"/>
      <addaction name="actionSaveAs"/>
      <addaction name="separator"/>
      <addaction name="actionSetting"/>
      <addaction name="actionExit"/>
     </widget>
     <widget class="QMenu" name="menuHelp">
      <property name="title">
       <string/>
      </property>
      <addaction name="actionAbout"/>
     </widget>
     <widget class="QMenu" name="menuView">
      <property name="title">
       <string/>
      </property>
      <addaction name="actionTreeViewEnable"/>
      <addaction name="separator"/>
      <addaction name="actionTreeViewShowAll"/>
      <addaction name="actionTreeViewCloseAll"/>
      <addaction name="actionTreeViewClean"/>
     </widget>
     <widget class="QMenu" name="menu">
      <property name="title">
       <string/>
      </property>
      <addaction name="actionUndo"/>
      <addaction name="actionRedo"/>
      <addaction name="separator"/>
      <addaction name="actionCut"/>
      <addaction name="actionCopy"/>
      <addaction name="actionPaste"/>
      <addaction name="separator"/>
      <addaction name="actionFind"/>
     </widget>
     <addaction name="menuFile"/>
     <addaction name="menu"/>
     <addaction name="menuView"/>
     <addaction name="menuHelp"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QTreeView" name="treeView">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOpen">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionTreeViewEnable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>启用左侧的文件列表</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionTreeViewShowAll">
   <property name="text">
    <string/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionTreeViewCloseAll">
   <property name="text">
    <string/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionTreeViewClean">
   <property name="text">
    <string/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
